### ISSUE TITLE
mlar doesn't load institutions

### ISSUE THREAD

**Issue by omnipresent07** [cfpb/hmda-frontend#31]
Short description explaining the high-level reason for the new issue.

## Steps to replicate behavior (include URLs)

1.

## Screenshots
<img width="965" alt="Screen Shot 2019-12-12 at 7 25 20 AM" src="https://user-images.githubusercontent.com/44377678/70711912-9f42d300-1cb0-11ea-9da3-d7b5f2767e5e.png">

**Comment by meissadia** [comment]
Content Security Policy error.

Should be fixed once Issue #21 is merged and deployed.

**Comment by meissadia** [comment]
Fixed

![mlar-leis](https://user-images.githubusercontent.com/2592907/70745201-8b16ca00-1ce0-11ea-8e81-1c4cb8b63888.png)
